src/toUrl.js

```diff
@@ -23,6 +23,10 @@ export default (to?: ?To, routesMap: RoutesMap): string => {
       return basename ? basename + path : path
     }
     catch (e) {
+      if (action.type === NOT_FOUND) {
```

Comment:
I think this logic should live in `actionToPath`

Reply:
yea, so we gotta stop--because RFR proper won't be getting updated besides the new next big release. 

i dont want you to have to waste any more of your time. i need to find the time to holistically look at this. at the end of day we're just trying to hide a warning. i appreciate u taking the time to make this library as ill as possible. my best recs for u are:

- monkeypatch console.warn to not show these errors
- conditionally show a different element for your search button's disabled state

it's all good brotha. ur ocd like me. let's let this one hang. nobody else is having problems with this. i'll take into consideration all ur effort here when i can invest time into holistically revisiting this package.